### DISCUSSION TITLE
Best Links and Resources about Next.js

### DISCUSSION THREAD

**Discussion by agustif** [vercel/next.js#10641] [Help] [answered]
# Next.js list of links and resources.

## Repo

[NextJS Repository](https://github.com/zeit/next.js)
(You're Here on its discussion tab, inside a discussion.)

## Docs

[NextJS Docs](https://nextjs.org/docs)

## Showcase

[Showcase Examples using NextJS](https://nextjs.org/showcase)
[Companies / Sites using NextJS disccussion](https://github.com/zeit/next.js/discussions/10640)

## Blog

[Official Blog](https://nextjs.org/blog/)

### blogposts

[Next.js 9.2](https://nextjs.org/blog/next-9-2)
[Next.js 9.1.7](https://nextjs.org/blog/next-9-1-7)

### BASICS

[Getting Started](https://nextjs.org/learn/basics/getting-started)
[Navigate Between Pages](https://nextjs.org/learn/basics/navigate-between-pages)
[Using Shared Components](https://nextjs.org/learn/basics/using-shared-components)
[Create Dynamic Pages](https://nextjs.org/learn/basics/create-dynamic-pages)
[Clean URLs with Dynamic Routing](https://nextjs.org/learn/basics/clean-urls-with-dynamic-routing)
[Fetching Data for Pages](https://nextjs.org/learn/basics/fetching-data-for-pages)
[Styling Components](https://nextjs.org/learn/basics/styling-components)
[API Routes](https://nextjs.org/learn/basics/api-routes)
[Deploying a Next.js App](https://nextjs.org/learn/basics/deploying-a-nextjs-app)

### EXCEL

[Export into a Static HTML App](https://nextjs.org/learn/excel/static-html-export)
[TypeScript](https://nextjs.org/learn/excel/typescript)
[Lazy Loading Modules](https://nextjs.org/learn/excel/lazy-loading-modules)
[Lazy Loading Components](https://nextjs.org/learn/excel/lazy-loading-components)
[Create AMP Pages](https://nextjs.org/learn/excel/amp)
[Automatic Static Optimization](https://nextjs.org/learn/excel/automatic-static-optimization)

---
P.S: This might somewhat be redundant to awesome-nextjs I guess. but I'm gonna go ahead anyway, It'll be useful for me myself at least, hope it is also useful for the community.

Please do share your links and I'll do my best to keep the index/first post updateed

**Comment by leerob** [comment]
I've also put together some links here ➡️https://masteringnextjs.com/resources

**Comment by Vadorequest** [comment, marked as answer]
For people looking for a production-grade boilerplate, with Zeit and TypeScript, take a look at https://github.com/UnlyEd/next-right-now

It can really help you get started much much faster, or be used as a reference and learning resource.

**Reply by leerob** [reply]
This is really awesome 😍

**Reply by Vadorequest** [reply]
Thank you! It's the result of months of hard work to put all those moving pieces together.

It's also very much documented to help anyone implement parts of it on their own basecode. Sharing is caring!
